R/tm_a_pca.R

```diff
@@ -238,72 +238,65 @@ ui_a_pca <- function(id, ...) {
           )
         )
       ),
-      forms = teal::get_rcode_ui(ns("rcode")),
+      forms = teal.widgets::verbatim_popup_ui(ns("rcode"), "Show R code"),
       pre_output = args$pre_output,
       post_output = args$post_output
     )
   )
 }
 
-srv_a_pca <- function(id, datasets, reporter, dat, plot_height, plot_width, ggplot2_args) {
+srv_a_pca <- function(id, data, reporter, filter_panel_api, dat, plot_height, plot_width, ggplot2_args) {
```

Comment:
Empty filter renders a red error:
![Screenshot from 2022-09-02 13-31-32](https://user-images.githubusercontent.com/86970066/188130652-aa03b087-f1d2-4b08-94b6-c8f8c6617e2c.png)

Reply:
I'm on it